revpi_core.c

```diff
@@ -47,15 +47,6 @@ static const struct kthread_prio revpi_core_kthread_prios[] = {
 	{ }
 };
 
-static struct gpiod_lookup_table revpi_core_gpios = {
-	.dev_id = "piControl0",
-	.table  = { GPIO_LOOKUP_IDX("pinctrl-bcm2835", 42, "Sniff1A", 0, GPIO_ACTIVE_HIGH),	// 1 links
```

Comment:
Please document the pin names (1A, 1B, ...) in a code comment in the device-tree so that the information doesn't get lost when you delete it here.